packages/x-data-grid/src/components/panel/filterPanel/GridFilterInputBoolean.tsx

```diff
@@ -81,7 +82,8 @@ function GridFilterInputBoolean(props: GridFilterInputBooleanProps) {
           label={label}
           value={filterValueState}
           onChange={onFilterChange}
-          variant="standard"
+          variant={variant}
+          notched={variant === 'outlined' ? true : undefined}
```

Comment:
@mui/xgrid Now that I look at this, I realized that the outlined variant of MUI TextField is actually a better choice for UX, because it looks much more like a text input than the standard variant. What do we think about using the outlined variant by default?

Reply:
I don't if this comment is for me. Basically, I think that it is better to have standard by default because all the examples are using this variant.